daemon/emer-main.c

```diff
@@ -291,8 +291,6 @@ on_bus_acquired (GDBusConnection *system_bus,
   g_autofree gchar *tracking_id = NULL;
 
   tracking_id = emer_daemon_get_tracking_id (daemon);
```

Comment:
These are now unused and can be removed.

Reply:
ok